### PR TITLE
fixing dimension report alert

### PR DESCRIPTION
**Pull request by puneetjaiswal**
- fixed anomaly dashboard link when #anomalies are zero
- added sorting of dimensions based on first time bucket value
- todo: color shading of the value cells

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/AlertTaskRunner.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.linkedin.thirdeye.anomaly.alert;
 
+import com.linkedin.pinot.pql.parsers.utils.Pair;
```
Comment (kishoreg) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/AlertTaskRunner.java]: use Immutable Pair from Apache Commons. Avoid using internal data structures from Pinot

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/AlertTaskRunner.java [hunk 2]
```diff
@@ -286,18 +291,38 @@ private void sendAlertForAnomalies(String collectionAlias, List<MergedAnomalyRes
...
           currentDimension = report.getResponseData().getResponseData().get(index)[dimensionIndex];
           valueMap.put(String.valueOf(report.getTimeBuckets().get(q).getCurrentStart()), String
-              .format("%+.1f", report.getResponseData().getResponseData().get(index)[valIndex]));
+              .format("%+.1f", Double.valueOf(report.getResponseData().getResponseData().get(index)[valIndex])));
         }
-        dimensionValueMap.put(currentDimension, valueMap);
+        dimensionValueList.add(new Pair<>(currentDimension, valueMap));
       }
+
+      Collections.sort(dimensionValueList, (o1, o2) -> {
```
Comment (kishoreg) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/AlertTaskRunner.java]: is this ok with java 1.7, lets make sure code compiles with 1.7 as well

Comment (puneetjaiswal) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/AlertTaskRunner.java]: I will make the change to make it compatible with java 7

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/AlertTaskRunner.java (hunk 1 above)
Comment (puneetjaiswal) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/AlertTaskRunner.java]: Sure. This came as auto import.

**Pull request merged**